server/view.go

```diff
@@ -375,13 +450,14 @@ var pageSource = `<!DOCTYPE html>
 						{{- end}}
 						{{- if not .Recording}}
 						<div class="code-area">
-							<textarea readonly>{{.Code}}</textarea>
+							<textarea readonly>replay {{.Code}}</textarea>
 						</div>
 						{{- end}}
 					</div>
 					<footer class="card-footer">
 						{{- if not .Recording}}
 						<a class="card-footer-item" onclick="copyCode(this)">Copy to clipboard</a>
+						<a class="card-footer-item" href="/watch?id={{.GameID}}">Spectate</a>
```

Comment:
Game IDs are not guaranteed to be unique across regions. Please include the platform as well.